packages/gatsby-plugin-preload-fonts/README.md

```diff
@@ -54,6 +54,10 @@ npm run preload-fonts -- --no-sandbox
 
 See [Puppeteer](#puppeteer) for more information.
 
+### Using with CI
+
+If you're planning to use this plugin in CI you can set environment variable `CI`. Please note that in this case `font-preload-cache.json` will only get rebuild when routes change.
```

Comment:
```suggestion
If you're planning to use this plugin in CI you can set the environment variable `CI`. Please note that in this case `font-preload-cache.json` will only get rebuild when routes change.
```
I think it would be helpful to give an example of using the environment variable. Is it just a flag? Does it have a value?

Reply:
From what I've seen in the code isCI utility checks for name and if it doesn't match anything than it will accept anything. I guess that `CI=true` would be suitable for example. Please tell me what you think about it

Reply:
That sounds great.

Reply:
Updated ;)

Reply:
I'm pretty sure all CI environments will set the `CI` env var automatically